compute_endpoint/globus_compute_endpoint/engines/globus_compute.py

```diff
@@ -34,17 +33,17 @@ def __init__(
         self.run_dir = os.getcwd()
         self.label = label
         self._status_report_thread = ReportingThread(
-            target=self.report_status, args=[], reporting_period=heartbeat_period
+            target=self.report_status,
+            args=[],
```

Comment:
This will collapse to a single line if you drop the last comma.

Reply:
Done